<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlashDialog</class>
 <widget class="QDialog" name="FlashDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>334</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Flash Image</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/org.firebird-emus.firebird-emu.png</normaloff>:/icons/resources/org.firebird-emus.firebird-emu.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Diags:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelDiags">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="buttonSave">
       <property name="text">
        <string>Save as..</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Boot2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBoot2">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBoot2">
       <property name="text">
        <string>Select file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Manuf:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelManuf">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDiags">
       <property name="text">
        <string>Select file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonManuf">
       <property name="text">
        <string>Select file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_7">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>OS:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelOS">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOS">
       <property name="text">
        <string>Select file</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Calculator Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QComboBox" name="selectModel">
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string notr="true">Touchpad</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">Touchpad CAS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">CX</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">CX CAS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">CX II (/-T/CAS)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="selectModelRev">
       <item>
        <property name="text">
         <string notr="true">HW-A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">HW-J (CXCR)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string notr="true">HW-W (CXCR4)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
